deepmd/tf/model/model.py

```diff
@@ -144,7 +144,7 @@ def __init__(
         self.data_stat_protect = data_stat_protect
         self.srtab_name = use_srtab
         if self.srtab_name is not None:
-            self.srtab = PairTab(self.srtab_name)
+            self.srtab = PairTab(self.srtab_name, rcut=sw_rmax)
```

Comment:
should not be `sw_rmax`, which defines the maximum of the potential switching region.

Reply:
Does it still contribute energy out of `sw_rmax`?